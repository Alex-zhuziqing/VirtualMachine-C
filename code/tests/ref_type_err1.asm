FUNC LABEL 0
    REF REG 0 VAL 8
    RET
