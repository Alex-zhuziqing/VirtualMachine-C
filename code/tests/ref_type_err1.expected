REF cannot have the second argument of the types VAL or REG
